templates/results-row.php

```diff
@@ -14,5 +14,10 @@
 	<td>
 		{{data.message}}
 	</td>
+	<# if ( data.link ) { #>
+		<td>
+			<a href="{{data.link}}" aria-label="<?php esc_attr_e( 'View file in the plugin file editor.', 'plugin-check' ); ?>" target="_blank"><?php esc_html_e( 'View in code editor', 'plugin-check' ); ?></a>
```

Comment:
What is the benefit of an `aria-label` here over the regular link anchor text? Is that needed for accessibility? I may not be up to date with the requirements there, so it would be great if you could clarify.

I am concerned about the `_target="blank"` here from an accessibility perspective. If we want to keep it, we should make it obvious that this link opens in another window/tab with relevant screen reader text.

Reply:
Thanks for bringing this to my attention. I have removed the `aria-label` attribute and added screen reader text in https://github.com/WordPress/plugin-check/pull/298/commits/5df8da4698ccff1d0998f8435cbae17ee8a91435, following the approach used in WordPress.